virtcontainers/sandbox.go

```diff
@@ -1386,7 +1392,30 @@ func (s *Sandbox) UpdateContainer(containerID string, resources specs.LinuxResou
 		return err
 	}
 
-	return c.update(resources)
+	newResources := ContainerResources{
+		VCPUs: uint32(utils.ConstraintsToVCPUs(*resources.CPU.Quota, *resources.CPU.Period)),
+		// do page align to memory, as cgroup memory.limit_in_bytes will be aligned to page when effect.
+		// TODO use GetGuestDetails to get the guest OS page size.
+		MemByte: (*resources.Memory.Limit >> 12) << 12,
```

Comment:
@clarecch  should I move this to `calcHotplugMemMiBSize()` ?

Reply:
I don't think we have need